fcos-config.yaml

```diff
@@ -46,7 +46,33 @@ storage:
         name: core
       target: /usr/lib/systemd/user/podman.socket
       hard: false
+    - path: /home/core/.config/systemd/user/default.target.wants/cockpit-core-user.service
+      user:
+        name: core
+      group:
+        name: core
+      target: /home/core/.config/systemd/user/cockpit-core-user.service
+      hard: false
   files:
+      # Add systemd unit for the cockpit bearer token
+    - path: /home/core/.config/systemd/user/cockpit-core-user.service
+      mode: 0755
```

Comment:
Does this need to be executable? the default permissions are probably good enough?